bundles/com.espressif.idf.ui/plugin.xml

```diff
@@ -98,6 +98,11 @@
             id="com.espressif.idf.ui.command.eclipeguide"
             name="%command.name.9">
       </command>
+      <command
```

Comment:
Could we remove icons that are not getting used? Windows and Linux, etc

Reply:
removed